gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
Any reason to stay so low? 4.10.2 is latest

Reply:
that's just the current "least" version for build-tools 3.2.0.